org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/extensions/XMLExtensionsRegistry.java

```diff
@@ -197,6 +197,12 @@ void unregisterExtension(IXMLExtension extension) {
 		extensions.remove(extension);
 		extension.stop(this);
 	}
+	
+	public void disposeExtensions() {

```

Comment:
Please add javadoc which explains it unregister extension and please rename this method to dispose only because in the future we could dispose other things than extension.

Reply:
Okay will do